content/assignments/memory-allocator/allocator/helpers.h

```diff
@@ -0,0 +1,33 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+
+#pragma once
+
+#include <errno.h>
+#include <limits.h>
+#include <stddef.h>
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <sys/mman.h>
+#include <unistd.h>
+
+#define DIE(assertion, call_description)						\
```

Comment:
Use the `DIE()` in the `utils/` folder to also get rid of the `checkpatch.pl` errors.